server/src/test/java/org/elasticsearch/common/recycler/AbstractRecyclerTestCase.java

```diff
@@ -43,6 +43,11 @@ public void destroy(byte[] value) {
             Arrays.fill(value, DEAD);
         }
 
+        @Override
+        public int pageSize() {
+            return 10;
```

Comment:
nit: suggest ` byte[] value = new byte[pageSize()];` in `newInstance()` to avoid the magic number 10